core/src/test/java/org/apache/iceberg/TestLocationProvider.java

```diff
@@ -114,7 +114,7 @@ public void testDefaultLocationProvider() {
   @Test
   public void testDefaultLocationProviderWithCustomDataLocation() {
     this.table.updateProperties()
-        .set(TableProperties.WRITE_NEW_DATA_LOCATION, "new_location")
+        .set(TableProperties.WRITE_FOLDER_STORAGE_LOCATION, "new_location")
```

Comment:
Does it make sense to leave a test for the deprecated config value?

I’d love to even log something if a user sets it, but that can be considered later (if at all).

Reply:
It is the same value, just using a different name to avoid confusion.

Reply:
Yeah I see now. This won't affect any external folks unless they have some custom stuff that touches this config.